### PR TITLE
setup-linux: Use -y with apt commands

### PR DESCRIPTION
**Pull request by vigneshvg**
Some apt install/update commands could get stuck waiting on yes/no
confirmation. Always use -y with apt to ensure that we do not get
stuck.

### CODE REVIEW

**Review by wantehchang — APPROVED** [review]
Please wait for Vincent's approval.

Vincent: I think we can merge this pull request before the v1.1.1 release.

File: .github/actions/setup-linux/action.yml
Comment (wantehchang) [inline .github/actions/setup-linux/action.yml:40]: @vigneshvg @y-guyon We are using brew and vcpkg on Mac and Windows. I wonder if they have a similar option.

I searched for "-y" and "yes" in the [brew man page](https://docs.brew.sh/Manpage) but didn't find anything. Same for the [vcpkg install doc page](https://learn.microsoft.com/en-us/vcpkg/commands/install#options).

Comment (vigneshvg) [reply .github/actions/setup-linux/action.yml:40]: > @vigneshvg @y-guyon We are using brew and vcpkg on Mac and Windows. I wonder if they have a similar option.
> 
> I searched for "-y" and "yes" in the [brew man page](https://docs.brew.sh/Manpage) but didn't find anything. Same for the [vcpkg install doc page](https://learn.microsoft.com/en-us/vcpkg/commands/install#options).

yeah i was just searching for it as well. it seems like bre and vcpkg assume -y by default and they don't ask for any user prompts.

**Comment by vrabaud** [comment]
Sure, let's merge. have we seen cases on the CI where it actually freezes?

**Pull request merged**
